packages/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode, useState } from 'react'
+
+import { Divider, Item, List } from './styled'
+
+interface AccordionItemProps {
+  children?: ReactNode
+  className?: string
+}
+/* istanbul ignore next */
+export function useAccordionController() {
+  const [activeItem, setActiveItem] = useState<number>()
+
+  const handleItemClick = (index: number) => {
+    setActiveItem((prev) => (prev === index ? undefined : index))
+  }
+
+  return { activeItem, handleItemClick }
+}
+
+export function AccordionItem({ children, className }: AccordionItemProps) {
+  return (
+    <Item className={className}>
+      {children}
+      <Divider />
+    </Item>
+  )
+}
+
+export default List
+export { List as Accordion }
```

Comment:
Optional: Could we move these exports below the interface since they are the main exports of this file, to get some sort of hierarchy?